.github/workflows/release.yml

```diff
@@ -12,6 +11,9 @@ jobs:
       - name: Check Out Repo
         uses: actions/checkout@v2
 
+      - uses: gacts/github-slug@v1 # Action page: <https://github.com/gacts/github-slug>
+        id: slug
```

Comment:
This is my own GitHub action for the version extracting. Forget about `echo ${GITHUB_REF##*/} | sed -e 's/^[vV ]*//'` forever 😄